### DISCUSSION TITLE
2007/04/22/%E6%95%B0%E6%8D%AE%E6%B5%81%E9%A9%B1%E5%8A%A8%E7%9A%84%E7%BC%96%E7%A8%8B%E8%AF%AD%E8%A8%80

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#154] [Announcements]
# 2007/04/22/%E6%95%B0%E6%8D%AE%E6%B5%81%E9%A9%B1%E5%8A%A8%E7%9A%84%E7%BC%96%E7%A8%8B%E8%AF%AD%E8%A8%80

在面向对象的编程思想出现以前，文本编程语言主要采用的是面向过程的编程方法。面向过程有时候也被称为控制流驱动的编程方法。想我以前编程序，经常要先设计一个流程图，然后，照着流程图翻译成 C 代码就行了。

https://blog.qizhen.xyz/2007/04/22/%E6%95%B0%E6%8D%AE%E6%B5%81%E9%A9%B1%E5%8A%A8%E7%9A%84%E7%BC%96%E7%A8%8B%E8%AF%AD%E8%A8%80

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Unknown
2007年07月29日 16:39 编辑
阮先生:
      您好,我想向您请教一个问题: 我现在在用labview 做测试系统,但想结合agent 的思想把整个测试系统作成多agent的形式(也就是说分成驱动agent、测试agent、数据管理agent等），请问，labview有这样的功能吗？我是不是应该先用其他的软件（如java等）先编写，然后在labview中合成呢？ 

回复
Qizhen
2007年04月30日 23:20 编辑
当然不能完全替代C或者java，每个语言都有自己的特色，是是和应用的场合。在某种情况下，使用LabVIEW很方便，但另一些情况就是使用其它语言更适合。如果那种语言能够被完全替代，它的生命就结束了，你也就看不到它了。LabVIEW 不是万能的，做不了的事情，比做的了的要多:)

回复
liang
2007年04月29日 22:33 编辑
感觉楼主说的很有道理，但是想问下，LabVIEW到底是否可以完全替换C和JAVA代码？
感觉有的地方不能实现啊？

回复
杰
2007年04月27日 21:40 编辑
这个对于理解LabVIEW很有用，期待你早点写完。
呵呵，正在拜读。
LV中，race condition很容易就会出现。虽然它是即时编译的，也无法避免。

回复
Carol
2007年04月27日 18:07 编辑
好奇问一下，您的专业是什么？应该是计算机吧？！不是HRM的吧？！知识涉猎面真广
想来在企业做管理信息系统一定没问题

回复
竹贤
2007年04月26日 14:31 编辑
写下去啊。

回复
xia
2007年04月24日 07:17 编辑
很少见人 写space写得那么认真地

回复
arleie
2007年04月23日 21:29 编辑
专业不同
   不是很懂。。不过看起来很认真。
      有时间多来你这学习学习。。

回复
晨恺
2007年04月23日 16:09 编辑
如果添加那就更好了,可惜你限制了.哎

回复
Vivien
2007年04月23日 14:58 编辑
前辈,还是那句话,您适合出本书或者做讲师
什么时候来吉大我一定去听,虽然专业南辕北辙 哈哈
虾仁的点击和朋友数~

回复
